src/workspace.rs

```diff
@@ -411,3 +429,170 @@ impl Workspace {
         f(root_manifest_path)
     }
 }
+
+/// Subset of cargo profile settings to configure defaults for building contracts
+pub struct Profile {
+    opt_level: OptLevel,
+    lto: Lto,
+    // `None` means use rustc default.
+    codegen_units: Option<u32>,
+    overflow_checks: bool,
+    panic: PanicStrategy,
+}
+
+impl Default for Profile {
+    fn default() -> Profile {
+        Profile {
+            opt_level: OptLevel::One,
+            lto: Lto::Bool(false),
+            codegen_units: None,
+            overflow_checks: false,
+            panic: PanicStrategy::Unwind,
```

Comment:
Not sure in whichever context unwinding is applicable to Wasm smart contracts. But I still like it being here as reference. Obviously those defaults are not applicable anyways, right?

Reply:
I copied the `Profile` struct originally from [`cargo`](https://github.com/rust-lang/cargo/blob/b6a4b074ed924878fefb631c7d559ca47331e5eb/src/cargo/core/profiles.rs#L584), and brought the `Default` impl with it. It might be useful for reference but I think I will remove it since it is unused.

As far as having `Unwind` as a possible value of `PanicStrategy`, although it is not applicable I think it is useful to keep it since it is a valid value for that field.

Reply:
Maybe emit a warning if a user tries to compile an ink! contract for the Wasm target (`target=wasm32-unknown-unknown`) and with `panic=unwind`.